odd_collector_azure/adapters/azure_sql/mappers/models.py

```diff
@@ -0,0 +1,35 @@
+from typing import Any, NamedTuple
+from dataclasses import dataclass
+
+
+@dataclass
+class TableMetadata:
```

Comment:
Do we need it if you have `odd_collector_azure/adapters/azure_sql/domain/table.py` ?
Decide and leave something one

Reply:
Done